### DISCUSSION TITLE
dev Branch的文档页面中，“帮助改善此页面”按钮定位到不存在的Branch

### DISCUSSION THREAD

**Discussion by Eden7Ba23** [YunYouJun/hexo-theme-starry#171] [General 大局为重]
刚刚准备给有关Waline的代码示范中加入有关DarkMode的内容，
但是”第三方支持“下方的”帮助改善此页面“定位到的不存在的Branch"main"，如下图所示：
![](https://pic.edenjohnson.cyou/images/202109212010441.png)

**Comment by YunYouJun** [comment]
稍等，我修改一下，实际是 `dev` 分支。以及 Waline 我来添加一下默认 dark 支持。（

**Reply by YunYouJun** [reply]
Done.
